ink-wrapper/Cargo.toml

```
@@ -12,3 +12,5 @@ clap = { version = "4.1", features = ["derive"] }
 anyhow = "1.0"
 genco = "0.17"
 hex = "0.4.3"
+ink_metadata = "4.0.1"
+scale-info = "2.3"
```

Can we fix these versions? i.e.
```suggestion
ink_metadata = "=4.0.1"
scale-info = "=2.3"
```
I don't trust anymore that releases will follow SemVer correctly - better safe than sorry.

---

Fair, will do.